editor/src/java/com/defold/libs/ResourceUnpacker.java

```diff
@@ -104,11 +105,21 @@ public static void unpackResources() throws IOException, URISyntaxException {
                 if (alreadyUnpacked) {
                     logger.info("Already unpacked for the editor version {}", sha1);
                 } else {
-                    unpackResourceFile("builtins.zip", unpackPath.resolve("builtins"));
+                    unpackResourceFile("lib/builtins.zip", unpackPath.resolve("builtins"), false, true);
                     unpackResourceDir("/_unpack", unpackPath);
                     unpackResourceFile("libexec/" + platform.getPair() + "/libogg" + platform.getLibSuffix(), unpackPath);
                     unpackResourceFile("libexec/" + platform.getPair() + "/liboggz" + platform.getLibSuffix(), unpackPath);
-                    unpackResourceFile("libexec/" + platform.getPair() + "/oggz-validate" + platform.getExeSuffixes()[0], unpackPath, true);
+                    unpackResourceFile("libexec/" + platform.getPair() + "/oggz-validate" + platform.getExeSuffixes()[0], unpackPath, true, false);
+
+                    if (Platform.matchPlatformAgainstOS(platform.getPair(), OS.OS_ID_WINDOWS)) {
```

Comment:
On Windows `dmengine` and libs are the same for the editor as for bob (because symbols are in a separate `dmengine.pdb` file), so we don't need these files in unpack folder and can take it from there

Reply:
dmengine.exe is part of bob.jar in libexec/x86_64-win32/dmengine.exe and from the editor you have it in `defold-83536c65275c04366b704b2fd3f3d681c88b8267.jar`(replace wih "your" sha1) in `_unpack/x86-64_win32/bin/dmengine.exe`.

Reply:
`dmengine.exe` is in `libexec` folder
![CleanShot 2024-12-29 at 20 34 10@2x](https://github.com/user-attachments/assets/a66aac6f-587b-470f-a2cf-59051942ecf1)

Reply:
We had two equal `dmengine.exe` in two different places. So, now the editor takes one from the same place bundler takes it (It's only for windows, because on windows exe files are equal and symbols are in `dmengine.pdb`, on the other platforms symbols are included into the executable)

Reply:
I see, thanks! I will use `libexec`.